addons/mail/static/src/chatter/web/chatter.xml

```diff
@@ -96,7 +96,7 @@
                 <Composer composer="state.thread.composer" autofocus="true" className="state.composerType === 'message' ? '' : 'pt-4'" mode="'extended'" onPostCallback.bind="onPostCallback" dropzoneRef="rootRef" type="state.composerType" t-key="props.threadId"/>
             </t>
         </div>
-        <div class="o-mail-Chatter-content">
+        <div class="o-mail-Chatter-content d-flex flex-column flex-grow-1">
```

Comment:
chatter diff?

Reply:
Yes, forgot to update PR comment, commit had it: it centers the empty text in chatter when aside, otherwise the text is close to chatter topbar and it looked off.
<img width="2554" alt="Screenshot 2024-09-23 at 17 53 08" src="https://github.com/user-attachments/assets/9feb177d-8ac1-42b2-85c0-dc173da3e5f4">

In practice most chatter have at least "Someone created the new record" message, so they are usually not empty.
There's plan to remove some useless mail.message in DB notably this one, so improving the empty screen will make sense even in chatter.